### DISCUSSION TITLE
grad_accum_steps Vits

### DISCUSSION THREAD

**Discussion by SyGen899** [coqui-ai/TTS#2186] [General Q&A] [answered]
Hi,
I was hoping someone can help me out, Im trying to train a VITS model, and I read that gradient accumulation was setup on the trainer API, so I specified grad_accum_steps to 4, using a batch size of 16. but when I run training my step size remains the same? not sure if I'm missing something here in the config to use this? or if I'm misunderstanding something here, any help appreciated. :)

**Comment by erogol** [comment, marked as answer]
Gradient accumulation does not work with VITS or any GAN models currently due to the gradient caching needed between generator and discriminator steps.

**Reply by SyGen899** [reply]
thanks for the response.
